plugin-flex-ts-template-v2/src/feature-library/teams-view-columns/flex-hooks/components/TeamsViewColumns.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import * as Flex from '@twilio/flex-ui';
+
+import { Worker } from '../../../../types/task-router';
+import { FlexComponent } from '../../../../types/feature-loader';
+
+interface WorkerItem {
+  worker: Worker;
+}
+
+const getSkills = (item: WorkerItem) => {
+  return item.worker.attributes.routing ? item.worker?.attributes?.routing?.skills?.join(' / ') : 'NONE';
```

Comment:
what are your thoughts on this being a ", " instead of a " / "?  i dont feel strongly but at first glance "," it might be more conventional.